src/http/Message.h

```diff
@@ -143,9 +144,11 @@ class Message : public RefCountable
 
 } // namespace Http
 
-inline void
-HTTPMSGUNLOCK(Http::Message *a)
+template <class T>
+void
```

Comment:
I know it is not required here, but let's leave `inline` after converting this function into a template because we (still) want the compiler to inline this (now templated) function. Squid is not very consistent about the application of the "write inline when you mean it" [rule of thumb](https://stackoverflow.com/questions/10535667/does-it-make-any-sense-to-use-inline-keyword-with-templates), but that rule is better than nothing, and a lot of Factory code in Squid does have the inline in similar contexts.

Reply:
Done.